pattern-library/html/boilerplate-blank.html

```diff
@@ -0,0 +1,51 @@
+<!--
+edX Pattern Library: Boilerplate HTML Template: Header + Footer
+
+NOTES:
+
+* base template with details on how to handle primary/secondary assets, metadata, and basic view-level UI (on canvas vs. conditional and off-canvas elements)
+-->
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8">
+        <!-- always force latest IE rendering engine or request chrome rrame -->
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
```

Comment:
If you're aiming for strict validation here, it's best to define these in the headers for the document, not in the page source itself.  IE=edge,chrome=1 is not a valid value for the content attribute here.